addons/html_editor/static/src/others/dynamic_placeholder_plugin.js

```diff
@@ -74,13 +74,20 @@ export class DynamicPlaceholderPlugin extends Plugin {
      * @param {string} chain
      * @param {string} defaultValue
      */
-    onValidate(chain, defaultValue) {
+    async onValidate(chain, defaultValue) {
         if (!chain) {
             return;
         }
 
+        const fields = await this.services.orm.call(this.defaultResModel, 'fields_get', [['partner_id', chain]], {});
+        const partnerField = fields['partner_id'];
+        const chainField = fields[chain];
+
```

Comment:
```suggestion
        const { partner_id: partnerField, [chain]: chainField } = await this.services.orm.call(
            this.defaultResModel,
            "fields_get",
            [["partner_id", chain]],
            {}
        );

```

Reply:
Hello @mrsr-odoo 
Requested changes are done! Can you please have a look at it.
Please let me know if I have missed something.
Thanks :smile: